docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -158,11 +158,17 @@ For more information, see the link:https://github.com/zchee/docker-machine-drive
 To use Minishift with Hyper-V:
 
 . Install link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[Hyper-V].
+
+. Add the user to the local Hyper-V Administrators group.
++
+NOTE: These permissions are needed to allow the user to create and delete virtual machines with the Hyper-V Management API. 
+For more information, see xref:../roubleshooting/troubleshooting-driver-plugins.adoc#hyper-v-commands-must-be-run-as-an-administrator[Hyper-V commands must be run as an Administrator]
+
 . Add an link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network[External Virtual Switch].
 +
 NOTE: Verify that you pair the virtual switch with a network card (wired or wireless) that is connected to the network.
 
-. If you have multiple virtual switches, set the environment variable `HYPERV_VIRTUAL_SWITCH` to the name of the external virtual switch you want to use for Minishift.
+. Set the environment variable `HYPERV_VIRTUAL_SWITCH` to the name of the external virtual switch you want to use for Minishift.
```

Comment:
So, now it is compulsory to use this environment variable, right? I found here https://github.com/minishift/minishift/pull/1367#pullrequestreview-62339143

Reply:
mandatory, compulsory, needed, required... ;-)
until we have a better solution.

Reply:
Since: https://github.com/minishift/minishift/pull/1162/files#diff-183dcc0860348e8238c68dcd04fc87c0R219